src/components/dashboard/Dashboard.js

```diff
@@ -73,7 +73,7 @@ import FaceVerificationError from './FaceVerification/screens/ErrorScreen'
 import FaceVerificationUnsupported from './FaceVerification/screens/UnsupportedScreen'
 import FaceVerificationDuplicateError from './FaceVerification/screens/DuplicateErrorScreen'
 import FaceVerificationPortraitModeError from './FaceVerification/screens/PortraitModeErrorScreen'
-import FaceVerificationOtherError from './FaceVerification/screens/OtherErrorScreen'
```

Comment:
1. Remove unsupported screen. FaceVerificationUnrecoverableError will be used instead
2. Restore "Other error". It's a different screen we should also have
3. We have an VerificationError component was re-usable. As all error screens are too different, we couldn't reuse it anymore. So delete VerificationError please
4. The main idea was to have **single** `ErrorScreen` which would display different content depending to the error type (for camera issue it would return null but open popup dialog). Please, move those screens to verification components and remoive them from dashboard navigation. Other logic of displaying corresponding errors i'll perform by myself

Reply:
5. I saw you made an Clipboard error screen. It's not related to this task. Please move it to the separated PR attached to the #1790

Reply:
done